src/Microsoft.AspNet.Server.Kestrel/compiler/preprocess/SPCM.cs

```diff
@@ -1,6 +0,0 @@
-﻿namespace Microsoft.AspNet.Server.Kestrel
-{
-    public class StandardsPoliceCompileModule : Microsoft.StandardsPolice.StandardsPoliceCompileModule
```

Comment:
So we are no longer getting any warning from standards police after this is removed, but I see the issue you are running into. Having any .cs file in the compiler/preprocess folder is causing dnu build to fail with a null ref violation.

@davidfowl Any idea why this is happening now?